### ISSUE TITLE
Obsolete warning: "The attribute may be dropped by IE7"

### ISSUE THREAD

**Issue by 1jj** [validator/validator#1642]
`<a href>` causes this warning:
```
Attribute href without an explicit value seen. The attribute may be dropped by IE7.
```
This warning should be omitted without replacement (later IE versions are not affected).

Thanks!
